README.md

```diff
@@ -305,6 +305,56 @@ in your system configuration and
 
 in your Home Manager configuration.
 
+Flakes
+------
+
+Home Manager includes a flake.nix for compatibility with [NixOS flakes](https://nixos.wiki/wiki/Flakes) that wish
+to use it as a module. A bare-minimum flake.nix would be as follows:
```

Comment:
```suggestion
Perhaps something like
```suggestion
Home Manager includes a `flake.nix` for compatibility with [NixOS flakes](https://nixos.wiki/wiki/Flakes) for those
that wish to use it as a module. A bare-minimum `flake.nix` would be as follows:
```
?